.github/workflows/build-docs-verifier.yml

```diff
@@ -1,7 +1,6 @@
 name: Build docs-verifier
 on:
-  pull_request_target:
-    types: [opened, synchronize, reopened]
+  pull_request
 
```

Comment:
`pull_request_target` caused the action to not run against the current branch, but on the `main` branch.